bugsnag-android-unity/src/main/java/com/bugsnag/android/unity/UnityClient.java

```diff
@@ -0,0 +1,56 @@
+package com.bugsnag.android.unity;
+
+import java.lang.String;
+
+import android.content.Context;
+
+import com.bugsnag.android.Bugsnag;
+import com.bugsnag.android.Callback;
+import com.bugsnag.android.Client;
+import com.bugsnag.android.Error;
+import com.bugsnag.android.MetaData;
+import com.bugsnag.android.Report;
+import com.bugsnag.android.Severity;
+
+public class UnityClient {
+
+    public static void init(Context androidContext, String apiKey) {
+        Bugsnag.init(androidContext, apiKey);
+    }
+
+    public static void notify(String name, String message, String context, StackTraceElement[] stacktrace, Severity severity, String logLevel) {
+        Bugsnag.notify(name, message, stacktrace, new UnityCallback(context, severity, logLevel));
+    }
+}
+
+class UnityCallback implements Callback {
```

Comment:
Could this top-level class be moved to a separate source file?

Reply:
Yeah 🤔 There's no reason the two classes need to be in the same file.